serverless/aws/lambda/src/main/scala/sttp/tapir/serverless/aws/lambda/runtime/AwsLambdaRuntime.scala

```diff
@@ -1,27 +1,26 @@
 package sttp.tapir.serverless.aws.lambda.runtime
 
-import cats.effect.{Blocker, ConcurrentEffect, ContextShift}
+import cats.effect.{ConcurrentEffect, ContextShift, Resource, Sync}
 import cats.syntax.all._
 import com.typesafe.scalalogging.StrictLogging
-import org.http4s.client.blaze.BlazeClientBuilder
-import sttp.client3.http4s.Http4sBackend
+import sttp.client3.Identity
+import sttp.client3.httpclient.HttpClientSyncBackend
 import sttp.tapir.server.ServerEndpoint
 import sttp.tapir.serverless.aws.lambda._
 
 import scala.concurrent.ExecutionContext
-import scala.concurrent.duration.DurationInt
 
-abstract class AwsLambdaRuntime[F[_]: ContextShift: ConcurrentEffect] extends StrictLogging {
-  def endpoints: Iterable[ServerEndpoint[_, _, _, Any, F]]
+abstract class AwsLambdaRuntime extends StrictLogging {
```

Comment:
we have to keep the `F` constraint here, as the endpoints we want to use should be parametrised with `F` as well. See this backend: https://sttp.softwaremill.com/en/latest/backends/fs2.html#using-httpclient-java-11